modules/ztp-removing-the-argocd-pipeline.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="ztp-removing-the-argocd-pipeline_{context}"]
+= Removing the ArgoCD pipeline
+
+Use the following procedure if you want to remove the ArgoCD pipeline and all generated artifacts.
+
+.Procedure
+
+. Detach all clusters from ACM.
+
+. Delete all SiteConfig and PolicyGenTemplate custom resources (CRs) from your Git repository.
```

Comment:
s/SiteConfig and PolicyGenTemplate/`SiteConfig` and `PolicyGenTemplate`

Reply:
changed